07-regression1.Rmd

```diff
@@ -123,7 +133,7 @@ This function takes two arguments:
 2. `size` (the number of observations/rows to be randomly selected/sampled)
 
 ```{r 07-sacramento}
-set.seed(1234)
+set.seed(2)
```

Comment:
@ttimbers because the original seed created a subset of data with a datapoint literally right at 2000 square feet (or at least, so close it wasn't possible to visually distinguish). The paragraph right before this says "we don't have data exactly at 2000 sqft". I changed the seed to get a different subset of data to work with that was more obviously not exactly at 2000.